server/src/main/java/org/candlepin/resource/OwnerProductResource.java

```diff
@@ -185,8 +188,10 @@ public Product getProduct(@PathParam("owner_key") String ownerKey,
     @Path("/{product_id}/certificate")
     @Produces(MediaType.APPLICATION_JSON)
     @SecurityHole
-    public ProductCertificate getProductCertificate(@PathParam("owner_key") String ownerKey,
-                                                    @PathParam("product_id") String productId) {
+    public ProductCertificate getProductCertificate(
+        @Verify(Owner.class) @PathParam("owner_key") String ownerKey,
+        @Verify(Product.class) @PathParam("product_id") String productId) {
```

Comment:
This one I almost think should go back to super admin only if it was in 1.0, which I suspect it was. This doesn't feel like something regular users or consumers should be calling, and I could tsee them having read access to the owner and product.